packages/checkout/widgets-lib/src/widgets/checkout/CheckoutWidget.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { CheckoutWidgetParams } from '@imtbl/checkout-sdk';
+import {
+  useContext,
+  useEffect,
+  useMemo,
+  useReducer,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import { ConnectLoaderContext } from '../../context/connect-loader-context/ConnectLoaderContext';
+import { CryptoFiatProvider } from '../../context/crypto-fiat-context/CryptoFiatProvider';
+import { EventTargetContext } from '../../context/event-target-context/EventTargetContext';
+import {
+  initialViewState,
+  SharedViews,
+  ViewContext,
+  viewReducer,
+} from '../../context/view-context/ViewContext';
+import { StrongCheckoutWidgetsConfig } from '../../lib/withDefaultWidgetConfig';
+import { ErrorView } from '../../views/error/ErrorView';
+import { LoadingView } from '../../views/loading/LoadingView';
+
+export type CheckoutWidgetInputs = CheckoutWidgetParams & {
+  config: StrongCheckoutWidgetsConfig,
+};
+
+export default function CheckoutWidget(props: CheckoutWidgetInputs) {
```

Comment:
@jwhardwick lets clean up this component to be empty, since this will just load the iframe, dont require any of the existing providers.

Elements we can keep are:
- config props
- checkout and provider instances

and just have it render an empty div.